back/config/routes.rb

```diff
@@ -77,6 +77,9 @@
       end
 
       resources :background_jobs, only: %i[index]
+      resources :tracked_jobs, only: %i[index show]
```

Comment:
A bit confused here, isn't it only `jobs` and not `tracked_jobs`, since I only see a `JobsController`?

Reply:
Oops, wrong move. I'll remove it.